vignettes/mlr3spatiotempcv.Rmd

```diff
@@ -63,55 +63,67 @@ Currently, the following resampling methods are implemented:
 
 To create a spatial task, use class `TaskClassifST` for classification and `TaskRegrST` for regression tasks.
 Please make sure that your dataset comes with coordinates since they are needed during the construction of the class (see the next code chunk).
-In addition, one can specify whether the coordinates should be used as features during modeling.
-<!-- coordinates are put into another object? Question is if is important for the user to know that xy-columns will be removed from the feature dataset if he/she already specified that coords should not be used as predictors.-->
-By default, coordinates are not used as features and will be removed from the feature dataset after the coordinates have been created.
+In addition, one can specify whether the coordinates should be used as features during the modeling.
+By default, coordinates are not used as features and will be removed from the feature dataset after the coordinates have been added as a method to the task.
 For optional plotting purposes, it is recommended to also pass the PROJ string specifying the used coordinate reference system via the `crs` argument.
 
-Printing a spatial task, also displays a preview of the coordinates.
-
 ```{r}
 data = mlr3::as_data_backend(readRDS(system.file("extdata", "ecuador.rda",
   package = "mlr3spatiotempcv")))
 
-TaskClassifST$new(id = "ecuador", backend = data, target = "slides",
+ecuador = TaskClassifST$new(id = "ecuador", backend = data, target = "slides",
   positive = "TRUE", coordinate_names = c("x", "y"), coords_as_features = FALSE,
   crs = "+proj=utm +zone=17 +south +datum=WGS84 +units=m +no_defs")
 ```
 
-After the task has been constructed, it can be used in conjunction with all available resampling methods.
-However, non-spatial methods should **only be used to showcase the degree of overoptimistic performance estimates** compared to a chosen spatiotemporal resampling method [@schratz2019].
+The "ecuador" dataset consists of a response variable named `slides` (boolean) and various predictor variables that might have an influence on the occurrence of landslides such as slope, curvature or catchment area.^[Please note that the "ecuador" task is also available via `tsk{ecuador}`.]
+For a detailed overview of the dataset, please refer to @brenning_landslide_2015 and/or Chapter 11 of [Geocomputation with R](https://geocompr.robinlovelace.net/spatial-cv.html) [@lovelace_geocomputation_2019].
 
-### Resampling
-
-To showcase the usage with code, we adapt the basic [resampling example of the mlr3book](https://mlr3book.mlr-org.com/resampling.html).
+<!-- maybe we can add a figure for illustration purposes, e.g., 11.1 from https://geocompr.robinlovelace.net/spatial-cv.html?-->
```

Comment:
Sure we can, feel free to!

Reply:
Ok, for now, I will cross-reference the png from the book here. Of course, we can also build the figure ourselves and even further improve it, e.g., by adding an inset map. But this would also mean, that we would need a few other dependencies (*tmap*, *raster*, *RSAGA*). Probably you don't want this?

Reply:
> But this would also mean, that we would need a few other dependencies (tmap, raster, RSAGA). Probably you don't want this?

Oh yes, I prefer to avoid these. 
Linking to the online PNG of the book is fine - this way, possible updated will automatically be pulled in.